pkg/azuredisk/controllerserver.go

```diff
@@ -280,6 +280,21 @@ func (d *Driver) ControllerPublishVolume(ctx context.Context, req *csi.Controlle
 		return nil, status.Error(codes.InvalidArgument, "Volume capability not supported")
 	}
 
+	diskName, err := getDiskName(diskURI)
```

Comment:
can you write a standalone function(`checkDiskExists`) in `azuredisk.go`?  I see `ValidateVolumeCapabilities` func will also use this function.

Reply:
Okay, I have made the function.